js/script.js

```diff
@@ -0,0 +1,62 @@
+// Create array of books
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const addBook = document.querySelector('#book-collection');
+const btn = document.querySelector('#add-button');
+
+if (!localStorage.getItem('bookInfo')) {
+  localStorage.setItem('bookInfo', JSON.stringify([]));
+}
+
+let books;
+
+function saveBooks(book) {
+  localStorage.setItem('bookInfo', JSON.stringify(book));
+}
+
+function displayBook() {
+  books = JSON.parse(localStorage.getItem('bookInfo'));
+  // eslint-disable-next-line no-use-before-define
+  userInterface();
+}
+displayBook();
+
+function removeBook() {
+  books.splice(this, 1);
```

Comment:
- [**Optional**] We shall practice using array methods that doesn't mutate the original array such as: [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of `push`, `pop`, `splice` and so on.instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on.

Reply:
Hi, @FrederickMih thanks for the feedback,